### PR TITLE
Update benchmark filter

### PR DESCRIPTION
**Pull request by hhaAndroid**
Update benchmark filter

### CODE REVIEW

**Comment by hhaAndroid** [comment]
1. fp16 not sure if it positions it correctly
2. not including  `fast_rcnn` , `legacy_1.x` and `scratch`

File: .dev_scripts/benchmark_filter.py [hunk 1]
```diff
@@ -27,74 +27,94 @@ def parse_args():
...
+    'configs/res2net/faster_rcnn_r2_101_fpn_2x_coco.py',
+    'configs/resnest/'
+    'mask_rcnn_s50_fpn_syncbn-backbone+head_mstrain_1x_coco.py',
+    'configs/retinanet/retinanet_r50_caffe_fpn_1x_coco.py',
+    'configs/rpn/rpn_r50_fpn_1x_coco.py',
     'configs/sabl/sabl_retinanet_r50_fpn_1x_coco.py',
+    'configs/scratch/mask_rcnn_r50_fpn_gn-all_scratch_6x_coco.py',
+    'configs/ssd/ssd300_coco.py',
+    'configs/tridentnet/tridentnet_r50_caffe_1x_coco.py',
+    'configs/wider_face/ssd300_wider_face.py',
```
Comment (ZwwWayne) [inline .dev_scripts/benchmark_filter.py:115]: No need

File: .dev_scripts/benchmark_filter.py [hunk 2]
```diff
@@ -27,74 +27,94 @@ def parse_args():
...
-    'configs/pascal_voc/faster_rcnn_r50_fpn_1x_voc0712.py',
-    'configs/pascal_voc/ssd300_voc0712.py',
+    'configs/nas_fcos/nas_fcos_nashead_r50_caffe_fpn_gn-head_4x4_1x_coco.py',
     'configs/nas_fpn/retinanet_r50_nasfpn_crop640_50e_coco.py',
-    'configs/mask_rcnn/mask_rcnn_r50_caffe_fpn_mstrain-poly_1x_coco.py',
-    'configs/gn+ws/mask_rcnn_r50_fpn_gn_ws-all_2x_coco.py',
-    'configs/gfl/gfl_r50_fpn_1x_coco.py',
     'configs/paa/paa_r50_fpn_1x_coco.py',
+    'configs/pafpn/faster_rcnn_r50_pafpn_1x_coco.py',
+    'configs/pascal_voc/retinanet_r50_fpn_1x_voc0712.py',
```
Comment (ZwwWayne) [inline .dev_scripts/benchmark_filter.py:101]: no need

File: .dev_scripts/benchmark_filter.py [hunk 3]
```diff
@@ -27,74 +27,94 @@ def parse_args():
...
+    'centripetalnet_hourglass104_mstest_16x6_210e_coco.py',
+    'configs/cityscapes/mask_rcnn_r50_fpn_1x_cityscapes.py',
+    'configs/cornernet/'
+    'cornernet_hourglass104_mstest_8x6_210e_coco.py',  # special
     'configs/dcn/mask_rcnn_r50_fpn_mdconv_c3-c5_1x_coco.py',
     'configs/dcn/faster_rcnn_r50_fpn_dpool_1x_coco.py',
     'configs/dcn/faster_rcnn_r50_fpn_mdpool_1x_coco.py',
     'configs/dcn/mask_rcnn_r50_fpn_dconv_c3-c5_1x_coco.py',
-    'configs/libra_rcnn/libra_faster_rcnn_r50_fpn_1x_coco.py',
+    'configs/deepfashion/mask_rcnn_r50_fpn_15e_deepfashion.py',
```
Comment (ZwwWayne) [inline .dev_scripts/benchmark_filter.py:64]: no need.

File: .dev_scripts/benchmark_filter.py [hunk 4]
```diff
@@ -27,74 +27,94 @@ def parse_args():
...
+    'configs/point_rend/point_rend_r50_caffe_fpn_mstrain_1x_coco.py',
+    'configs/regnet/mask_rcnn_regnetx-3GF_fpn_1x_coco.py',
+    'configs/reppoints/reppoints_moment_r50_fpn_gn-neck+head_1x_coco.py',
+    'configs/res2net/faster_rcnn_r2_101_fpn_2x_coco.py',
+    'configs/resnest/'
+    'mask_rcnn_s50_fpn_syncbn-backbone+head_mstrain_1x_coco.py',
+    'configs/retinanet/retinanet_r50_caffe_fpn_1x_coco.py',
+    'configs/rpn/rpn_r50_fpn_1x_coco.py',
     'configs/sabl/sabl_retinanet_r50_fpn_1x_coco.py',
+    'configs/scratch/mask_rcnn_r50_fpn_gn-all_scratch_6x_coco.py',
```
Comment (ZwwWayne) [inline .dev_scripts/benchmark_filter.py:112]: No need.

File: .dev_scripts/benchmark_filter.py [hunk 5]
```diff
@@ -27,74 +27,94 @@ def parse_args():
...
     'configs/faster_rcnn/faster_rcnn_r50_caffe_fpn_1x_coco.py',
-    'configs/empirical_attention/faster_rcnn_r50_fpn_attention_1111_dcn_1x_coco.py',  # noqa
-    'configs/reppoints/reppoints_moment_r50_fpn_gn-neck+head_1x_coco.py',
-    'configs/guided_anchoring/ga_faster_r50_caffe_fpn_1x_coco.py',
-    'configs/free_anchor/retinanet_free_anchor_r50_fpn_1x_coco.py',
-    'configs/fsaf/fsaf_r50_fpn_1x_coco.py',
-    'configs/scratch/mask_rcnn_r50_fpn_gn-all_scratch_6x_coco.py',
-    'configs/pafpn/faster_rcnn_r50_pafpn_1x_coco.py',
+    'configs/faster_rcnn/faster_rcnn_r50_caffe_fpn_mstrain_1x_coco.py',
+    'configs/faster_rcnn/faster_rcnn_r50_caffe_c4_1x_coco.py',
```
Comment (ZwwWayne) [inline .dev_scripts/benchmark_filter.py:74]: no need

**Comment by hhaAndroid** [comment]
Simple example：

1. run benchmark_filter.py to  generate benchmark json file, output:

```
{"models": ["configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py", "configs/retinanet/retinanet_r50_caffe_fpn_1x_coco.py"]}
```

2. run convert_benchmark_script.py to generate benchmark script sh file, output:

```
echo 'configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py' &

GPUS=8  GPUS_PER_NODE=8  CPUS_PER_TASK=2 ./tools/slurm_train.sh xxx faster_rcnn_r50_fpn_1x_coco configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py ./tools/faster_rcnn_r50_fpn_1x_coco >/dev/null &

echo 'configs/retinanet/retinanet_r50_caffe_fpn_1x_coco.py' &

GPUS=8  GPUS_PER_NODE=8  CPUS_PER_TASK=2 ./tools/slurm_train.sh xxx retinanet_r50_caffe_fpn_1x_coco configs/retinanet/retinanet_r50_caffe_fpn_1x_coco.py ./tools/retinanet_r50_caffe_fpn_1x_coco >/dev/null &
```

**Review by ZwwWayne — APPROVED** [review]

File: .dev_scripts/convert_benchmark_script.py [hunk 6]
```diff
@@ -33,28 +40,34 @@ def main():
...
-            command_info += stdout_cfg + ' &'
-
-            f.write(command_info)
-
-            if i < len(model_cfgs):
-                f.write('\n')
+    commands = []
+    for i, cfg in enumerate(model_cfgs):
+        # print cfg name
+        echo_info = 'echo \'' + cfg + '\' &'
```
Comment (ZwwWayne) [inline .dev_scripts/convert_benchmark_script.py:46]: Do not need & for echo infor?

Comment (hhaAndroid) [reply .dev_scripts/convert_benchmark_script.py:46]: & will be appended to every command

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**